config/examples/Creality/Ender-4/Configuration.h

```diff
@@ -803,11 +798,11 @@
  * When changing speed and direction, if the difference is less than the
  * value set here, it may happen instantaneously.
  */
-//#define CLASSIC_JERK
+#define CLASSIC_JERK
 #if ENABLED(CLASSIC_JERK)
-  #define DEFAULT_XJERK 10.0
-  #define DEFAULT_YJERK 10.0
-  #define DEFAULT_ZJERK  2.4
+  #define DEFAULT_XJERK 20.0
+  #define DEFAULT_YJERK 20.0
+  #define DEFAULT_ZJERK  0.4
 
```

Comment:
Junction Deviation is now the standard, so why revert to Classic Jerk? Also, keep in mind that Classic Jerk settings were cut in half back in Marlin 1.1.8.

Reply:
Classic Jerk may be better in-general for Cartesians. It's still an open question.